cdap-docs/integrations/source/ambari/index.rst

```diff
@@ -0,0 +1,278 @@
+
+#. If you wish to use the CDAP Explore service (to use SQL to query CDAP data), you must: have
+   Hive installed on the cluster; have the Hive client on the same host as CDAP; and set the
+   ``explore.enabled`` option to true.
+
+   .. figure:: ../_images/integration-ambari/ss07-config-enable-explore.png
+      :figwidth: 100%
+      :width: 800px
+      :align: center
+      :class: bordered-image
+ 
+      **Ambari Dashboard:** Enabling *CDAP Explore*
+
+   For a complete explanation of these options, refer to the :ref:`CDAP documentation on
+   cdap-site.xml <appendix-cdap-site.xml>`. After making any configuration changes, click
+   *Next*.
+
+#. Review the desired service layout and click *Deploy* to begin installing CDAP.
+
+   .. figure:: ../_images/integration-ambari/ss08-review-deploy.png
+      :figwidth: 100%
+      :width: 800px
+      :align: center
+      :class: bordered-image
+ 
+      **Ambari Dashboard:** Summary of Services
+
+#. Ambari will install CDAP and start the services.
+
+   .. figure:: ../_images/integration-ambari/ss09-install-start-test.png
+      :figwidth: 100%
+      :width: 800px
+      :align: center
+      :class: bordered-image
+ 
+      **Ambari Dashboard:** Install, Start, and Test
+      
+#. After the services are installed and started, you will click *Next* to get to the
+   Summary screen.
+
+#. This screen shows a summary of the changes that were made to the cluster. No services
+   should need to be restarted following this operation.
+
+   .. figure:: ../_images/integration-ambari/ss10-post-install-summary.png
+      :figwidth: 100%
+      :width: 800px
+      :align: center
+      :class: bordered-image
+ 
+      **Ambari Dashboard:** Summary
+
+#. Click *Complete* to complete the CDAP installation.
+
+#. Now, you should see **CDAP** listed on the main summary screen for your cluster.
+
+   .. figure:: ../_images/integration-ambari/ss11-main-screen.png
+      :figwidth: 100%
+      :width: 800px
+      :align: center
+      :class: bordered-image
+ 
+      **Ambari Dashboard:** Selecting *CDAP*
+
+#. Selecting *CDAP* from the left, or choosing it from the Services drop-down menu, will take
+   you to the CDAP service screen.
+
+   .. figure:: ../_images/integration-ambari/ss12-cdap-screen.png
+      :figwidth: 100%
+      :width: 800px
+      :align: center
+      :class: bordered-image
+ 
+      **Ambari Dashboard:** *CDAP* Service Screen
+ 
+Congratulations! CDAP is now running on your cluster, managed by Ambari.
+
+
+Roadmap and Future Features
+===========================
+CDAP integration with Ambari is still evolving and improving. Additional features are
+planned for upcoming versions of the CDAP Ambari Service, including a full smoke test of
+CDAP functionality after installation, pre-defined alerts for CDAP services, CDAP
+component High-availability support, select CDAP metrics, support for Kerberos-enabled
+clusters, and integration with the CDAP Authentication Server.
```

Comment:
I created issues for all of these... CDAP-4105 through CDAP-4110, if you want to link them to actual issues.